draft-ietf-scitt-architecture.md

```diff
@@ -950,8 +951,8 @@ A Relying Party SHOULD validate a Transparent Statement originating from a given
 Issuers cannot be stopped from producing Signed Statements including false assertions in their Statement payload (either by mistake or by corruption), but these Issuers can made accountable by ensuring their Signed Statements are systematically registered at a Transparency Service.
 
 Similarly, providing strong residual guarantees against faulty/corrupt Transparency Services is a SCITT design goal.
-Preventing a Transparency Service from registering Signed Statements that do not meet its stated Registration Policy, or to issue Receipts that are not consistent with their Append-only Log is not possible.
-In contrast Transparency Services can be held accountable and they can be called out by any Auditor that replays their Append-only Log against any contested Receipt.
+Preventing a Transparency Service from registering Signed Statements that do not meet its stated Registration Policy, or to issue Receipts that are not consistent with their Verifiable Data Structure is not possible.
+In contrast Transparency Services can be held accountable and they can be called out by any Auditor that replays their Verifiable Data Structure against any contested Receipt.
```

Comment:
```suggestion
In contrast Transparency Services can be held accountable and they can be called out by any Auditor that replays their Append-only log against any contested Receipt.
```

I don't think it's possible to do a full replay with just the digests, a replay from Signed Statements is necessary. As in other comments, I am perhaps being insufficiently imaginative when thinking about the VDS (which is a Merkle Tree in my mind).

Reply:
This is actually suggesting "full audit" of the log material is the only way to hold the TS to account. Its true that this can be done, i don't think its true that this is the only or best way to accomplish this. 

```suggestion
In contrast Transparency Services can be held accountable and they can be called by Auditors for any contested Receipt. An Auditor in posession of a complete sequence of registered signed statements can perform a full audit. Auditors who have verified the state of the log prior to the Receipts issuance, can use consistency proofs from that state to more efficiently resolve the dispute.
```

Reply:
Consistency proofs only work if what's disputed is consistency though, not correct registration checks. Should we separate the two kinds of audit?

Reply:
@achamayou, are you ok to commit the above change, and tweak? Or, do you have an alternative suggestion that we can commit?

Reply:
I am not ok with the second change, because it claims that consistency proofs are sufficient to resolve a dispute. This is not true, they cannot confirm whether registration was performed correctly, and although they can be used to detect forks, they cannot be used to decide which fork(s) contain malicious content. I have amended my proposed change to remove the mention of "Append-only log" and use the term "Verifiable Data Structure", in an effort to find common ground.